### PR TITLE
Rename MasterPlaylistController to PlaylistController

### PR DESCRIPTION
**Pull request by gkatsev**

### CODE REVIEW

File: design/media-groups.md [hunk 1]
```diff
@@ -9,10 +9,10 @@ These are used to represent:
 
 Currently, [MediaGroups][mg] interacts with several different classes, which can be difficult to follow and has cause bugs in the past. These components are:
 - multiple [DashPlaylistLoaders][dpl] or [PlaylistLoaders][pl]
-- [MasterPlaylistController][mpc]
+- [PlaylistController][pc]
 - multiple [SegmentLoaders][sl]
 
-[MediaGroups][mg] are **setup** by the `loadedmetadata` handler for the `masterPlaylistLoader` (video playlist loader) of a source; contained in the [MasterPlaylistController][mpc] (MPC). The [MPC] will pass in an `audioSegmentLoader` and `subtitleSegmentLoader` to be shared by the `AUDIO` and `SUBTITLE` mediaGroups respectively.
+[MediaGroups][mg] are **setup** by the `loadedmetadata` handler for the `masterPlaylistLoader` (video playlist loader) of a source; contained in the [PlaylistController][pc] (pc). The [pc] will pass in an `audioSegmentLoader` and `subtitleSegmentLoader` to be shared by the `AUDIO` and `SUBTITLE` mediaGroups respectively.
```
Comment (misteroneill) [inline design/media-groups.md:15]:
```suggestion
[MediaGroups][mg] are **setup** by the `loadedmetadata` handler for the `masterPlaylistLoader` (video playlist loader) of a source; contained in the [PlaylistController][pc] (PC). The [PC] will pass in an `audioSegmentLoader` and `subtitleSegmentLoader` to be shared by the `AUDIO` and `SUBTITLE` mediaGroups respectively.
```

**Review by misteroneill — COMMENTED** [review]
One small suggestion, but otherwise LGTM.

File: design/media-groups.md (hunk 1 above)
Comment (gkatsev) [reply design/media-groups.md]: Updated in a separate commit

**Comment by gkatsev** [comment]
We should probably merge the other PRs into `next` first. I'll rebase this on top of them so that those PRs aren't a pain.

**Comment by gkatsev** [comment]
Merge with a BREAKING_CHANGE: footer about the renamed class/properties.

**Review by misteroneill — APPROVED** [review]

**Pull request merged**